ethcore/src/state/account.rs

```diff
@@ -202,8 +208,8 @@ impl Account {
 			return Ok(value);
 		}
 		let db = SecTrieDB::new(db, &self.storage_root)?;
-
-		let item: U256 = db.get_with(key, ::rlp::decode)?.unwrap_or_else(U256::zero);
+		let unwrapping_decoder = |bytes:&[u8]| ::rlp::decode(&bytes).unwrap_or_else(|_| U256::zero());
```

Comment:
I believe we should have `.expect` instead of `unwrap_or_else` here. If the value exists in the database, we always expect it to be encoded properly.